packages/teams-js/src/public/pages.ts

```diff
@@ -690,4 +690,55 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigateToSelf API
+   */
+  export interface NavigateToSelfParams {
+    /**
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
```

Comment:
Would be good to use @linkcode to link to the deprecated EntityId property on context